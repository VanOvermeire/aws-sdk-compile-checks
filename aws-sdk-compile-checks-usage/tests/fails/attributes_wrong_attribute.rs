use aws_sdk_compile_checks_macro::required_props;

#[required_props(fake)]
async fn do_call() {
    // irrelevant
}

fn main() {}
