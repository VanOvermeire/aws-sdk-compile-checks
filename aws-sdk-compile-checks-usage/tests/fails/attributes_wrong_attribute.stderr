error: the only allowed attribute is `sdk`
 --> tests/fails/attributes_wrong_attribute.rs:3:18
  |
3 | #[required_props(fake)]
  |                  ^^^^
